open_event/static/css/admin/scheduling.css

```diff
@@ -0,0 +1,177 @@
+/*
+
+    Layers (top to bottom)
+        Session Element - 2
+        Session Holder - 1
+        Timeline - -1
+
+
+*/
+
+.sessions-holder {
+    z-index: 1 !important;
+}
+
+.session {
+    background-color: #efefef;
+    border: solid #b6b6b6 1px;
+    color: #3d3d3d;
+    height: 48px;
+    overflow: hidden;
+    width: 216px;
+	white-space: normal;
+    padding: 5px;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Using width with padding can sometimes make elements larger than you expect.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2650812118/issues/source?bid=3233864&fileBranchId=3324174#l23)